src/ert/run_models/everest_run_model.py

```diff
@@ -152,80 +101,50 @@ def __init__(
-        optimization_callback: OptimizerCallback,
-        display_all_jobs: bool = True,
+        simulation_callback: SimulationCallback | None,
+        optimization_callback: OptimizerCallback | None,
     ):
-        everest_config = self._add_defaults(everest_config)
-
         Path(everest_config.log_dir).mkdir(parents=True, exist_ok=True)
         Path(everest_config.optimization_output_dir).mkdir(parents=True, exist_ok=True)
 
-        self.ropt_config = everest2ropt(everest_config)
-        self.everest_config = everest_config
-        self.support_restart = False
+        assert everest_config.environment is not None
+        logging.getLogger(EVEREST).info(
+            "Using random seed: %d", everest_config.environment.random_seed
+        )
+        logging.getLogger(EVEREST).info(
+            "To deterministically reproduce this experiment, "
+            "add the above random seed to your configuration file."
+        )
+
+        self._ropt_config = everest2ropt(everest_config)
+        self._everest_config = everest_config
 
         self._sim_callback = simulation_callback
         self._opt_callback = optimization_callback
         self._fm_errors: dict[int, dict[str, Any]] = {}
-        self._simulation_delete_run_path = (
-            False
-            if everest_config.simulator is None
-            else (everest_config.simulator.delete_run_path or False)
-        )
-        self._display_all_jobs = display_all_jobs
         self._result: OptimalResult | None = None
-        self._exit_code: Literal["max_batch_num_reached"] | OptimizerExitCode | None = (
-            None
+        self._exit_code: EverestExitCode | None = None
+        self._evaluator_cache: _EvaluatorCache | None = (
+            _EvaluatorCache()
+            if (
+                everest_config.simulator is not None
+                and everest_config.simulator.enable_cache
+            )
+            else None
         )
-        self._max_batch_num_reached = False
-        self._simulator_cache: SimulatorCache | None = None
-        if (
-            everest_config.simulator is not None
-            and everest_config.simulator.enable_cache
-        ):
-            self._simulator_cache = SimulatorCache()
         self._experiment: Experiment | None = None
-        self.eval_server_cfg: EvaluatorServerConfig | None = None
-        storage = open_storage(config.ens_path, mode="w")
-        status_queue: queue.SimpleQueue[StatusEvents] = queue.SimpleQueue()
-        self.batch_id: int = 0
-        self.status: SimulationStatus | None = None
+        self._eval_server_cfg: EvaluatorServerConfig | None = None
+        self._batch_id: int = 0
+        self._status: SimulationStatus | None = None
 
         super().__init__(
             config,
-            storage,
+            open_storage(config.ens_path, mode="w"),
             config.queue_config,
-            status_queue,
+            queue.SimpleQueue(),
```

Comment:
Think in the next step we want to use this to forward the events from the base run model, so think it might be better to leave it as it was.